From 01d49fd1c64bbf77faae152b0f88ad0a96ed98e5 Mon Sep 17 00:00:00 2001
From: Oleg Karfich <oleg.karfich@wago.com>
Date: Fri, 7 Dec 2018 14:27:54 +0100
Subject: [PATCH] defconfig: pfc: enable bridge vlan filtering

This config is used by docker to virtualize bridges in container

Signed-off-by: Oleg Karfich <oleg.karfich@wago.com>
---
 arch/arm/configs/am3xxx_pfc_generic_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/am3xxx_pfc_generic_defconfig b/arch/arm/configs/am3xxx_pfc_generic_defconfig
index 4b7f9aa..defa849 100644
--- a/arch/arm/configs/am3xxx_pfc_generic_defconfig
+++ b/arch/arm/configs/am3xxx_pfc_generic_defconfig
@@ -202,6 +202,7 @@ CONFIG_BRIDGE_EBT_LOG=y
 CONFIG_L2TP=m
 CONFIG_L2TP_DEBUGFS=m
 CONFIG_BRIDGE=y
+CONFIG_BRIDGE_VLAN_FILTERING=y
 CONFIG_VLAN_8021Q=m
 CONFIG_VLAN_8021Q_GVRP=y
 CONFIG_NET_SCHED=y
-- 
2.7.4

